src/components/DrawerLayout.tsx

```diff
@@ -1,3 +1,6 @@
+/* eslint-disable eslint-comments/no-unlimited-disable */
+/* eslint-disable */
+// @ts-nocheck TODO(TS) provide types
 // @flow
```

Comment:
Why is flow used here?

Reply:
Swipeable and Drawer were typed using TS but with `// @flow` comment added. Not sure why, this will be removed after we migrate types.